pkgs/by-name/ra/razer-cli/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+  xrdb,
+}:
+
+# requires openrazer-daemon to be running on the system
+# on NixOS hardware.openrazer.enable or pkgs.openrazer-daemon
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "razer-cli";
+  version = "0-unstable-2024-04-04";
+
+  src = fetchFromGitHub {
+    owner = "lolei";
+    repo = pname;
+    rev = "e4c9a93f012fea2362e18c3374a38bc77a62d86b";
+    hash = "sha256-HWEYcYYHQs2uZZtiK+BaKU2yc90dJ+FEbR+ohq0Wkpw=";
+  };
+
+  propagatedBuildInputs = [
+    python3.pkgs.openrazer
+    xrdb
+  ];
```

Comment:
If these are supposed to be dependencies, then Python dependencies go into `dependencies`, and non-Python ones go under `buildInputs`